Dockerfile

```
@@ -0,0 +1,8 @@
+FROM quay.io/jupyter/minimal-notebook:afe30f0c9ad8
+
+COPY conda-linux-64.lock /tmp/conda-linux-64.lock
+
+RUN mamba create --name wine-quality-regressor --file /tmp/conda-linux-64.lock && \
```

I suggest doing the following command from the lectures because this will download the packages to the base conda environment of the container.

```suggestion
RUN mamba update  --quiet --file /tmp/conda-linux-64.lock && \
```